components/prism-autohotkey.js

```diff
@@ -1,16 +1,20 @@
 // NOTES - follows first-first highlight method, block is locked after highlight, different from SyntaxHl
 Prism.languages.autohotkey= {
-	'comment': {
-		pattern: /(^[^";\n]*("[^"\n]*?"[^"\n]*?)*)(?:;.*$|^\s*\/\*[\s\S]*\n\*\/)/m,
-		lookbehind: true
-	},
+	'comment': [
+		{
+			pattern: /(^|\s);.*/m,
+			lookbehind: true
+		},
+		{
+			pattern: /(^\s*)\/\*[^\n\r]*(?:(\n|\r|\r\n)(?:[^*]|\*[^\/])*(?:\*\/)?)?/m,
```

Comment:
How about this:

```js
/(^\s*)\/\*.*$(?:[\s\S]*?\*\/|[\s\S]*)/m
```

Like this, we can just let JS handle all different kinds of line breaks.